### PR TITLE
update PINCache

### PR DESCRIPTION
**Pull request by fumito-ito**
Older PINCache causes runtime erorr sometimes.
To avoid this problem, set newer version of PINCache.

refs: https://github.com/TextureGroup/Texture/pull/769

### CODE REVIEW

**Comment by dangthaison91** [comment]
Is there some new feature of new PinCache version you like to use @fumito-ito ? I not specified dependencies version so users can choose the version they want.

**Comment by fumito-ito** [comment]
@dangthaison91 No. According to TextureGroup/Texture#769, the runtime error `dyld: Symbol not found: _PINDiskCachePrefix` occurs when using Texture with Carthage. I also have experienced it in my project. To resolve this issue, we can use `PINCache 3.0.1-beta.6` or  later, not latest release version `2.6`.

File: Cartfile [hunk 1]
```diff
@@ -1,3 +1,3 @@
-
 github "RxSwiftCommunity/RxDataSources"
 github "texturegroup/texture"
+github "pinterest/PINCache" "3.0.1-beta.6"
```
Comment (dangthaison91) [inline Cartfile]: Can we allow user use version 3.0 or later by using `~> 1.0` or `>= 1.0`

**Review by dangthaison91 — CHANGES_REQUESTED** [review]

File: Cartfile (hunk 1 above)
Comment (fumito-ito) [reply Cartfile]: It will be better than mine. I fixed it.

**Review by dangthaison91 — APPROVED** [review]

**Pull request merged**